### ISSUE TITLE
how to get  testnet coin

### ISSUE THREAD

**Issue by xue960** [input-output-hk/cardano-sl#3083]
i  runing testnet node  and  what should  i do while i get some coin

**Comment by penuel-leo** [comment]
Yes, I use `connect-to-staging` started my testnet node, and Who or Where can i get some testnet ADA to test?

**Comment by refi93** [comment]
you mean this? http://testnet.iohkdev.io/goguen/faucet/

**Comment by guchenglin111** [comment]
https://github.com/input-output-hk/cardano-sl/issues/3647#issuecomment-424572542
